.env

```diff
@@ -45,6 +45,18 @@ MESSENGER_TRANSPORT_DSN_FAILED=doctrine://default?auto_setup=0
 CORS_ALLOW_ORIGIN='^https?://(localhost|127\.0\.0\.1)(:[0-9]+)?$'
 ###< nelmio/cors-bundle ###
 
+###> symfony/lock ###
+# Choose one of the stores below
+# postgresql+advisory://db_user:db_password@localhost/db_name
+LOCK_DSN=flock
+###< symfony/lock ###
+
+###> symfony/mailer ###
+MAILER_DSN=smtp://mailhog:1025
```

Comment:
We have switched over to mail-pit some time ago: https://github.com/itk-dev/devops_itkdev-docker/blob/develop/templates/symfony-6/docker-compose.yml#L69

Reply:
And renamed the service to mail, so if another one comes up... we don't have to change settings.